include/log_handling.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * \brief Function Prototype Library to implement log handling
+ * \author jaspalsingh@cmail.carleton.ca
+ * \version 1.0
+ * \date 2019-12-05
+ */
+
+#ifndef LOG_HANDLING_H
+#define LOG_HANDLING_H
+
+/**
+ * \brief Structure for storing log details
+ * \details Store log info such as log file pointer, log path, and log name
+ */
+struct log_info_t {
+    FILE *fptr;
+    char *log_filename;
+    char *log_basefolder;
+    char *log_path;
+};
```

Comment:
Same comment as my first code review, use typedef here, for example this would be:
```
typdef struct log_info_s {
    FILE *fptr;
    char *log_filename;
    char *log_basefolder;
    char *log_path;
} log_info_t;
```

Reply:
fixed in the latest commit to follow typedef